core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -1596,6 +1596,14 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.MASTER)
           .build();
+  public static final PropertyKey MASTER_EMBEDDED_JOURNAL_WRITE_REMOTE_ENABLED =
+      new Builder(Name.MASTER_EMBEDDED_JOURNAL_WRITE_REMOTE_ENABLED)
+          .setDefaultValue(false)
+          .setDescription("Whether the journal writer will write to remote master.")
```

Comment:
So currently, the leader will only write to local journal and Raft in the backend is responsible for replicating the leader local journal to all other masters?

During leadership transition, Raft will stop replicating the old leader journal to other masters and start replicating the new leader journal.

Is it true that all the local journal of the previous leading master will be replicated to all other nodes?
and the previous leading master steps down, it will fail to write to local journal because NoLeaderException and related RPCs will be returned as failure?

Reply:
Once stepped down, the journal writes in previous leader will fail and pending requests will be rejected. The clients will retry and eventually succeed once the new leader is up.